api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -132,6 +141,13 @@ type DRPlacementControlSpec struct {
 
 	// +optional
 	KubeObjectProtection *KubeObjectProtectionSpec `json:"kubeObjectProtection,omitempty"`
+
+	// Label selector to identify all the kube objects that need DR protection.
+	// It will be passed in to the VRG when it is created
+	// +optional
```

Comment:
optional here with the CEL rules allow an inital YAML that does not contain the field and then later a YAML apply that contains the field, based on the lastAppliedConfiguration (or so).

The pattern that we potentially want is to make this required, such that initial versions themselves set it or mark it empty.